packages/jest-stylelint-runner/package.json

```diff
@@ -23,10 +23,10 @@
   },
   "devDependencies": {
     "postcss": "8.4.23",
-    "stylelint": "14.16.1"
+    "stylelint": "15.10.3"
   },
   "peerDependencies": {
     "postcss": "8.x",
-    "stylelint": "14.x"
+    "stylelint": "15.x"
```

Comment:
This would be a breaking change. Do we want to do that now?
cc @CarlosCortizasCT

Reply:
Just saying that changing the required peer dependency version usually is considered a breaking change. I think Changeset will also reflect that (in the Versions packages PR).

Let's see maybe once we merge the PR, we can then still decide how to proceed.